source/NVDAObjects/__init__.py

```diff
@@ -308,6 +308,17 @@ def findOverlayClasses(self, clsList: typing.List[typing.Type["NVDAObject"]]) ->
 
 	beTransparentToMouse=False #:If true then NVDA will never consider the mouse to be on this object, rather it will be on an ancestor.
 
+	def objectFromPointRedirect(self, x, y):
+		"""Redirects NVDA to another object if this object is retrieved from on screen coordinates.
+		@param x: the x coordinate.
```

Comment:
The current style guide uses `:param:` I believe:
```suggestion
		:param x: the x coordinate.
```